pipelines/copy-number-variation-vector/gcp/CNVTasks.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+task ConsolidateHMMOutput {
+    meta {
+        description: "This task takes the output from the HMM subpipeline and combines it into a single tarball."
+    }
+    parameter_meta {
+        hmm_tarballs: "The output files from the HMM sub-pipeline. This is an array of tarballs, one for each sample."
+        output_dir: "The output directory for the tarball."
+    }
+    input {
+        Array[File] hmm_tarballs
+        String output_dir
+        # runtime values
+        String docker = "us.gcr.io/broad-gotc-prod/cnv:1.0.0-1679431881"
+        String ram = "8000 MiB"
+        Int cpu = 16
+        # TODO: Make disk space dynamic based on input size
```

Comment:
do you still want to do this?

Reply:
Changed this and the memory value to resize based on input size. Good catch!